src/main/scala/com/evolutiongaming/scaffeine/MetricsStatsCounter.scala

```diff
@@ -1,44 +1,48 @@
 package com.evolutiongaming.scaffeine
 
-import com.codahale.metrics.MetricRegistry
 import com.github.benmanes.caffeine.cache.stats.{CacheStats, StatsCounter}
-import nl.grons.metrics.scala.{MetricBuilder, MetricName}
+import io.prometheus.client.{CollectorRegistry, Counter}
 
 import scala.concurrent.duration.Duration
 
-class MetricsStatsCounter(registry: MetricRegistry, prefix: String) extends StatsCounter {
-  private val metrics = new MetricBuilder(MetricName(prefix), registry)
+class MetricsStatsCounter(registry: CollectorRegistry, prefix: String) extends StatsCounter {
 
-  private val misses = metrics.counter("misses")
+  private def createCounter(name: String): Counter = {
+    val c = Counter.build().name(s"${prefix}_$name".replaceAll("\\.", "_")).help(s"$prefix $name").create()
+    registry.register(c)
+    c
+  }
+
+  private val misses = createCounter("misses")
 
-  private val hits = metrics.counter("hits")
+  private val hits = createCounter("hits")
 
-  private val eviction = metrics.counter("evictions")
+  private val eviction = createCounter("evictions")
 
-  private val failure = metrics.counter("load_failure")
+  private val failure = createCounter("load_failure")
 
-  private val success = metrics.counter("load_success")
+  private val success = createCounter("load_success")
 
-  private val time = metrics.timer("load_time")
+  private val time = createCounter("load_time")
```

Comment:
Is it ok that we will have counter instead of timer for `load_time`?

Reply:
Here `Timer` directly was used just for summing durations.
Prometheus doesn't provide similar structure, except `Histogram.Timer`, which requires start/stop calls, but we get only durations here. Also, `Histogram` doesn't provide means to get a `count`, which we need here for statistics.
Only thing that is missing now are quantiles, though I do not see it as a big problem.